README.md

```diff
@@ -31,38 +31,48 @@
 
 <br>
 
+Strapi is a free and open-source framework with headless CMS capabilities for delivering your content anywhere you need.
```

Comment:
@alexandrebodin "free and open-source headless CMS made for" instead of "open-source framework with headless CMS capabilities for"